### PR TITLE
Update README with steps to run managers

### PR DESCRIPTION
**Pull request by amitmalav**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,55 +63,9 @@ Bosh Lite is a lightweight local development environment for BOSH using Warden/G
...
-* If you have installed bosh-lite using [bosh-deployment](https://github.com/cloudfoundry/bosh-deployment), then you can deploy cf using the following commands, using [cf-deployment](https://github.com/cloudfoundry/cf-deployment).
-
-```shell
-git clone https://github.com/cloudfoundry/cf-deployment
-cd cf-deployment
-bosh -e vbox upload-stemcell https://bosh.io/d/stemcells/bosh-warden-boshlite-ubuntu-trusty-go_agent
-bosh -e vbox update-cloud-config iaas-support/bosh-lite/cloud-config.yml
-bosh -e vbox -d cf deploy cf-deployment.yml -o operations/bosh-lite.yml --vars-store deployment-vars.yml -v system_domain=bosh-lite.com
-```
+To run Cloud Foundry with bosh 2.0, instructions from [this](https://starkandwayne.com/blog/running-cloud-foundry-locally-on-bosh-lite-with-bosh2/) article can be followed.
```
Comment (subhankarc) [inline README.md]: I would prefer to keep the actual texts, as the blogs may be moved/deleted.

File: README.md [hunk 2]
```diff
@@ -5,22 +5,23 @@
...
+1. [Local Development Setup](#local-development-setup-ubuntu)
+2. [Installing Docker](#installing-docker)
+3. [Installing NVM](#installing-nvm)
+4. [Installing Bosh Lite](#installing-bosh-lite)
+5. [Installing Cloud Foundry](#installing-cloud-foundry)
+6. [Installing the Broker](#installing-the-broker)
+7. [Launch the Broker](#launch-the-broker)
+8. [Register the Broker](#register-the-broker)
+9. [Launch the Managers](#launch-the-managers)
+10. [Upload Bosh Director Based Service Releases](#upload-bosh-director-based-service-releases)
```
Comment (subhankarc) [inline README.md]: Provide one more step as local APIServer setup and running of it, that would be a prerequisite right?

Comment (amitmalav) [reply README.md]: It is already mentioned below.
https://github.com/cloudfoundry-incubator/service-fabrik-broker/pull/370/files/b39dc1c443d5ef7e373a053f297098b952f327ea#diff-04c6e90faac2675aa89e2176d2eec7d8L184

File: README.md (hunk 1 above)
Comment (amitmalav) [reply README.md]: Okay. I'll create a wiki page for the same and link it here

File: README.md (hunk 2 above)
Comment (subhankarc) [reply README.md]: ah!...missed it.

File: README.md (hunk 1 above)
Comment (subhankarc) [reply README.md]: U dont like the description here? :)

Comment (subhankarc) [reply README.md]: better create a .md page and link here?

Comment (subhankarc) [reply README.md]: wiki is difficult to maintain version of.

File: README.md [hunk 3]
```diff
@@ -168,6 +126,7 @@ The required files mentioned above can be found here: https://github.com/cloudfo
 This process executes action scripts provided by services in restricted environment.
 More information on how to configure action scripts is documented here: https://github.com/cloudfoundry-incubator/service-fabrik-broker/wiki/Deployment-hooks-for-service-lifecycle-operations
 
+If predeployment actions are enabled in service catalog then deployment hook process has to be running.
```
Comment (ketakig12) [inline README.md]: Can you add if I do not want any actions to be executed, what should be done?

Comment (amitmalav) [reply README.md]: Added.

**Pull request merged**